### PR TITLE
OCPBUGS-766: Add warning message when the region/zone label value is 'Unnamed'

### PR DESCRIPTION
**Pull request by yanhua121**
Add warning message when the region/zone label value is 'Unnamed'

### CODE REVIEW

File: pkg/actuators/machine/reconciler.go [hunk 1]
```diff
@@ -316,7 +316,13 @@ func (r *Reconciler) updateMachineWithVMState(vm *nutanixClientV3.VMIntentRespon
 	}
 
 	vmRegion := *pcCluster.Spec.Name
+	if vmRegion == "Unnamed" {
+		klog.Warningf("%s: the machine.openshift.io/region label value is 'Unnamed', due to the prism-central cluster name is not set.", r.machine.Name)
```
Comment (elmiko) [inline pkg/actuators/machine/reconciler.go:320]: minor nit
```suggestion
		klog.Warningf("%s: the machine.openshift.io/region label value is 'Unnamed', due to the prism-central cluster name not being set.", r.machine.Name)
```

File: pkg/actuators/machine/reconciler.go [hunk 2]
```diff
@@ -316,7 +316,13 @@ func (r *Reconciler) updateMachineWithVMState(vm *nutanixClientV3.VMIntentRespon
...
+	}
 	vmZone := *vm.Status.ClusterReference.Name
+	if vmZone == "Unnamed" {
+		klog.Warningf("%s: the machine.openshift.io/zone label value is 'Unnamed', due to the prism-element cluster name is not set.", r.machine.Name)
```
Comment (elmiko) [inline pkg/actuators/machine/reconciler.go:324]: minor nit
```suggestion
		klog.Warningf("%s: the machine.openshift.io/zone label value is 'Unnamed', due to the prism-element cluster name not being set.", r.machine.Name)
```

**Review by elmiko — COMMENTED** [review]
just a grammar nit about the messages, otherwise looks good to me

File: pkg/actuators/machine/reconciler.go (hunk 1 above)
Comment (yanhua121) [reply pkg/actuators/machine/reconciler.go]: changed as suggested.

File: pkg/actuators/machine/reconciler.go (hunk 2 above)
Comment (yanhua121) [reply pkg/actuators/machine/reconciler.go]: changed as suggested.

**Comment by elmiko** [comment]
/retitle OCPBUGS-766 Add warning message when the region/zone label value is 'Unnamed'

**Comment by elmiko** [comment]
just trying to get the bot to pick up the changes

**Comment by elmiko** [comment]
/retitle OCPBUGS-766: Add warning message when the region/zone label value is 'Unnamed'

**Pull request merged**